src/plugins/chat.ts

```
@@ -2409,6 +2412,112 @@ function getReplyToolInputError(
+                    'video',
+                    'markdown'
+                ].includes(item.type)
+            ) {
+                return (
+                    `Field messages[${i}].content[${j}].type must be one of ` +
+                    'text, image, sticker, audio, at, face, voice, file, video, markdown'
+                )
+            }
+
+            if (
+                ['sticker', 'audio', 'file', 'video', 'voice', 'markdown'].includes(
```

ESLint

[error] Replace `'sticker',·'audio',·'file',·'video',·'voice',·'markdown'].includes(` with `⏎····················'sticker',⏎····················'audio',⏎····················'file',⏎····················'video',⏎····················'voice',`

(prettier/prettier)